tune_sklearn/_trainable.py

```diff
@@ -5,6 +5,7 @@
 from ray.tune import Trainable
 from sklearn.base import clone
 from sklearn.model_selection import cross_validate
+from sklearn.model_selection._validation import _aggregate_score_dicts
```

Comment:
Let's not depend on an internal implementation like this - https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/model_selection/_validation.py#L1606 

Can you move this out into our repo?